### PR TITLE
Adding prime actual weight 

### PR DESCRIPTION
**Pull request by kennethchow-truss**
[Jira Ticket](https://dp3.atlassian.net/browse/MB-11344)

## Summary

Added one line to accommodate the prime actual weight. Writing this PR and probably the writing the ticket probably took longer than making the change haha

## Setup to Run Your Code

1. Log in as a prime user
2. Navigate to move PENDNG
3. Click update shipment
4. Edit the actual pickup date and actual weight
5. Click save
6. Sign out 
7. Sign in as TIO user
8. Navigate to move PENDNG
9. Navigate to the move history tab
10. Observe that both actual pickup date and actual weight are created

### CODE REVIEW

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

File: src/constants/MoveHistory/Database/FieldMappings.js [hunk 1]
```diff
@@ -43,4 +43,5 @@ export default {
   receiving_agent: 'Receiving agent',
   releasing_agent: 'Releasing agent',
   tio_remarks: 'Max billable weight remark',
+  prime_actual_weight: 'Prime actual weight',
```
Comment (donaldthai) [inline src/constants/MoveHistory/Database/FieldMappings.js:46]: suggestion (blocking): I think the field name display text is suppose to be `Shipment weight` according to the [spreadsheet](https://docs.google.com/spreadsheets/d/1r-c01NLG1j2KgdZAnmiECs5e5goySIoEGw2Ue36DGK4/edit#gid=569262977&range=E28).

I would also append that field to the `LabeledDetails.test.jsx` file for [this test](https://github.com/transcom/mymove/blob/b1969b3d5cb3393eda91ce7114d4aaef791cdd6e/src/pages/Office/MoveHistory/LabeledDetails.test.jsx#L9). Would help to confirm what name and display text we're using.

**Review by donaldthai — CHANGES_REQUESTED** [review]

File: src/constants/MoveHistory/Database/FieldMappings.js (hunk 1 above)
Comment (donaldthai) [reply src/constants/MoveHistory/Database/FieldMappings.js:46]: another suggestion (blocking): Seems like the jira story is asking to add the `actual_pickup_date` field as display text `Departure date`

Comment (kennethchow-truss) [reply src/constants/MoveHistory/Database/FieldMappings.js]: Made the changes. Lemme know if there are other issues

**Review by donaldthai — APPROVED** [review]
⭐

**Pull request merged**